recipes-devtools/dmidecode/dmidecode/use-oe-vars.patch

```diff
@@ -0,0 +1,12 @@
+--- a/Makefile	2018-04-25 15:44:51.431420129 -0400
++++ b/Makefile	2018-04-25 15:36:35.327601950 -0400
+@@ -12,8 +12,7 @@
+ #   (at your option) any later version.
+ #
+ 
+-CC      = gcc
```

Comment:
Changing to ?= would allow our CC override to occur without breaking a normal build, if the patch is desired for upstreaming.

This dmidecode change should be in a separate commit with a rational.  Why do we need it?

Reply:
It's using the host machine's gcc when it should be using the Open Embedded gcc built through the toolchain (ie. i686-oe-linux-gcc)

Reply:
Right.  Does compiling with the host gcc cause a particular issue that i686-oe-linux-gcc avoids?

Reply:
I actually wrote this patch awhile ago, so I can't remember off the top of my head, but if you give me a few minutes to track it down I can definitely get an answer for ya.